examples/LED/LED.ino

```diff
@@ -52,9 +63,21 @@ void setup(void){
   adapter.addDevice(&led);
   adapter.begin();
   Serial.println("HTTP server started");
+  Serial.print("http://");
+  Serial.print(WiFi.localIP());
+  Serial.print("/things/");
+  Serial.println(led.id);
 }
 
 void loop(void){
   adapter.update();
-  digitalWrite(ledPin, ledOn.getValue().boolean ? LOW : HIGH); // active low led
+  status = ledOn.getValue().boolean;
```

Comment:
With the change at 52 status can be a local variable `on` here. `before` can be the only global variable